src/rnn/rnn_util.cpp

```diff
@@ -27,9 +27,19 @@
 #include <miopen/rnn.hpp>
 #include <miopen/rnn_util.hpp>
 #include <algorithm>
+#include <miopen/env.hpp>
+
+MIOPEN_DECLARE_ENV_VAR_UINT64(MIOPEN_RNNWRW_REDUCTION)
```

Comment:
[Recommendation] @shurale-nkn I suspect that
```C++
MIOPEN_DECLARE_ENV_VAR(MIOPEN_RNNWRW_REDUCTION, uint64_t, 1)
```
should define cached variable with default value 1.

Reply:
Ok, but in next commit or in next PR. Depending on CI result